src/payment/paypal.md

```diff
@@ -1,41 +1,57 @@
 ---
-title: Other PayPal Solutions
+title: PayPal Payment Solutions
 redirect_from:
   - /payment/paypal-all-in-one.html
+  - /payment/paypal-business-account.html
+  - /payment/paypal-payment-solutions.html
 ---
 
 {:.bs-callout-warning}
 **Payment Services Directive Requirements:** <br/>
-Beginning September 14, 2019, European banks may decline payments which do not meet [PSD2]({% link stores/compliance-payment-services-directive.md %}) requirements. For most PayPal solutions, no action is needed to comply with PSD2 because the requirements are handled by PayPal. For information about specific PayPal solutions, see the note at the top of each page.
+Beginning September 14, 2019, European banks may decline payments which do not meet [PSD2]({% link stores/compliance-payment-services-directive.md %}) requirements. For most PayPal solutions, no action is needed to comply with PSD2 because the requirements are handled by PayPal.
 
-PayPal is a global leader in online payments and a fast and secure way for your customers to pay online. With PayPal and Magento, you can accept payments from all major debit and credit cards. PayPal offers additional convenience without extra effort, because even your customers who don’t have a PayPal account can pay for their purchases with PayPal. In this guide, PayPal payments solutions are categorized as follows:
+PayPal is a global leader in online payments and a fast and secure way for your customers to pay online. The selection of available PayPal solutions varies by merchant location. PayPal Express Checkout and PayPal Payments Standard can be used in all parts of the world. To learn more, see [PayPal Solutions by Country]({% link payment/paypal-countries.md %}).
 
-- [Recommended Solutions (Paypal Express Checkout) ]({% link payment/paypal-express-checkout.md %})
-- PayPal All-In-One Payment Solutions
-- PayPal Payment Gateways
+## PayPal business account
 
-The selection of available PayPal solutions varies by merchant location. PayPal Express Checkout and PayPal Payments Standard can be used in all parts of the world. To learn more, see: [PayPal Solutions by Country]({% link payment/paypal-countries.md %}).
+To offer PayPal as a payment method in your store, you must have a PayPal [business account][1] and/or a PayPal Payflow account. The account requirements are specified in the description of each PayPal solution. Your PayPal merchant account is also used to manage any [fraud filters]({% link payment/paypal-fraud-management-filter.md %}) that are applied to purchases made from your store.
```

Comment:
is there a paypal payflow topic to link to?

Reply:
I'll look. The PayPal stuff is very convoluted from a product perspective.